Misc/NEWS.d/next/Windows/2020-05-30-02-46-43.bpo-38324.476M-5.rst

```diff
@@ -0,0 +1 @@
+Use wide-char _W_* fields of struct lconv on Windows
```

Comment:
```suggestion
Avoid Unicode errors when accessing certain locale data on Windows.
```

The NEWS entry is what people will read to see what's changed, so it helps to be applicable to users. Anyone who needs to know the technical details can find the commit.